sdk/apis/syncagent/v1alpha1/published_resource.go

```diff
@@ -68,6 +68,13 @@ type PublishedResourceSpec struct {
 	// many different kcp workspaces.
 	Naming *ResourceNaming `json:"naming,omitempty"`
 
+	// EnableClusterPaths toggles whether the Sync Agent will not just store the kcp
+	// cluster name as a label on each locally synced object, but also the full cluster
+	// path. This is optional because it requires additional requests to kcp and
+	// should only be used if the cluster path is of interest on the
+	// service cluster side.
+	EnableClusterPaths bool `json:"enableClusterPaths,omitempty"`
```

Comment:
Nitpick q: Should we be talking about "cluster paths" or "workspace paths" here? Because logical clusters don't really have paths, the path mapping happens through workspaces.

Reply:
I'm all yours regarding naming this flag. Within the codebase, I kinda liked the duality of `clusterName` and `clusterPath`, both variables have the same length and same prefix and just mhmhmhmhmhmhm chef's kiss, ya know?

Reply:
Let's go with workspace path please.